math/blis/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           github 1.0
+
+github.setup        flame blis 0.9.0
```

Comment:
please add "github.tarball_from releases/archive" whichever is appropriate

Reply:
@reneeotten Ok, sure.
But why is it needed though? Just to understand.

Reply:
But anyway, done.

Reply:
Tarball types "archive" and "tags" are both automatically generated by GitHub from any commitish/tag/branch. "tags" is the default for historical reasons but is deprecated by GitHub and is not recommended because it includes the author name in the extracted directory name. Projects often move from one author to another (e.g. when an author changes their GitHub handle or decides to move a project from their personal account to an organization account) which changes a "tags" tarball's checksums. This deficiency is corrected in the "archive" tarball type which is why it's recommended when a "releases" tarball is not available.